test/e2e/editors/autocompleteEditor.spec.js

```diff
@@ -2293,6 +2293,39 @@ describe('AutocompleteEditor', () => {
         done();
       }, 700);
     });
+
+    it('should allow render the html items without sanitizing the content', async() => {
+      const hot = handsontable({
+        columns: [
+          {
+            type: 'autocomplete',
+            source: [
+              '<b>foo <span>zip</span></b>',
+              '<i>bar</i><img src onerror="__xssTestInjection = true">',
+              '<strong>baz</strong>'
+            ],
+            allowHtml: true,
+          }
+        ]
+      });
+
+      selectCell(0, 0);
+      keyDownUp('enter');
+
+      await sleep(200);
+
+      const ac = hot.getActiveEditor();
+      const innerHot = ac.htEditor;
+
+      expect(window.__xssTestInjection).toBe(true);
+      expect(innerHot.getData()).toEqual([
+        ['<b>foo <span>zip</span></b>'],
+        ['<i>bar</i><img src onerror="__xssTestInjection = true">'],
+        ['<strong>baz</strong>'],
+      ]);
+
+      delete window.__xssTestInjection;
```

Comment:
What do you think about adding a scenario in `MemoryLeakTest`, where we can verify at the end if `window.__xssTestInjection` is not defined?

Reply:
I think that is not necessary. The variable depends on the case may or may not be defined by XSS type HTML. The `delete` ensures that the next test will have a clean environment.

But maybe it's a good idea to add the variable deletion after each test. WDYT?

Reply:
Maybe in common `afterEach` it might be overpowered. Single deletion in `autocompleteEditor` seems fine, maybe only in dropdownEditor we can add `afterEach` under `describe('allow html mode'...`.

Reply:
> Maybe in common afterEach it might be overpowered

Yeah, I agree. If there will be more tests of this type in the future, then I think it is worth to generalize the way how the variable is reset.

For now, I'll leave it as it is.